android/titanium/src/java/org/appcelerator/titanium/TiFileProxy.java

```diff
@@ -139,6 +138,13 @@ public boolean getWritable()
 	}
 
 	@Kroll.method
+	public boolean append(Object data)
```

Comment:
[Titanium.Filesystem.write](http://docs.appcelerator.com/platform/latest/#!/api/Titanium.Filesystem.File-method-write) should be exposed with the optional `append` parameter, so you can just make use of that 👍